tests/test_config_consistency.py

```diff
@@ -662,13 +665,28 @@ def get_runtime_metrics(agent):
     return runtime_metrics_gauges, runtime_metrics_sketches
 
 
-# Parse the JSON-formatted log message from stdout and return the 'dd' object
+# Parses the JSON-formatted log message from stdout and returns it
+# To pass tests that use this function, ensure your library has an entry in log_injection_fields
 def parse_log_injection_message(log_message):
```

Comment:
```suggestion
def parse_log_injection_message(log_message):
    # Parses the JSON-formatted log message from stdout and returns it
    # To pass tests that use this function, ensure your library has an entry in log_injection_fields
```
nit: in python we usually put the comment in the test 😅